app/uk/gov/hmrc/eoricommoncomponent/frontend/views/you_cant_use_service.scala.html

```diff
@@ -28,13 +28,11 @@
 @(userType: Option[AffinityGroup])(implicit messages: Messages, request: Request[_])
 
 @para1 = @{userType match {
-    case Some(Agent) => messages("cds.you-cant-use-service-agent.para1")
-    case _ => messages("cds.you-cant-use-service-standard-organisation.para1", longName)
+    case _ => messages("cds.you-cant-use-service-standard-organisation.para1")
```

Comment:
Since we now have only one outcome from this match statement, we can just remove it, no need to match if regardless of the outcome we always do the same thing 👍🏻  This is the same for @para2 below as well. We can just inline the messages to where theyre used, no need for variables.